data/user_data_base.ps2

```diff
@@ -199,6 +209,12 @@ function Get-MyName {
     return $nameTag.Value
 }
 
+function Get-Apps {
+    $instanceId = (New-Object System.Net.WebClient).DownloadString("http://169.254.169.254/latest/meta-data/instance-id")
+    $appsTag =  Get-EC2Tag | Where-Object {$_.ResourceId -eq $instanceId -and $_.Key -eq 'Apps'}
```

Comment:
The `Apps` tag dictates the value in `AppNames` in outpost2.config, as long as `AppNames` is null or empty, it'll be ignored by outpost2 and default to using hudl-{group}. But this gets me thinking there might be a subtle situation here with this approach of whitelisting the apps, mainly once this is non-empty, outpost2 will start filtering out apps according to what's specified, so for a hudl-queuedservices server for eg, in outpost2.config: `"AppNames": = ["hudl-queuedservices"]`, this will fail when we try to deploy outpost2 to this server as outpost2 isn't listed :( 

So if `Apps` is used, it really need to spell out all the apps that are going to be deployed to it, in this case, the `Apps` tag will be `alyx-outpost; hudl-queuedservices`, that creates `"AppNames": = ["alyx-outpost", "hudl-queuedservices"]`. I'm gonna test this out.